.github/workflows/integration.yml

```
@@ -92,6 +92,12 @@ jobs:
           fail_ci_if_error: true
           verbose: true
 
+      - name: Upload coverage information to coveralls
+        uses: coverallsapp/github-action@v2
+        with:
+          flag-name: Integration-${{ matrix.task }}
+          file: ./integration/integration-${{ matrix.task }}.txt
```

```suggestion
          file: ./integration/integration-${{ matrix.task }}.txt
          flag-name: integration-${{ matrix.task }}
```

To use the same order and lowercase flag names as in codecov.

The same in other places